backend/src/controller/meetupController.ts

```diff
@@ -314,4 +326,23 @@ export class MeetupController extends BaseController {
             return;
         });
     }
+
+    /**
+     * Notify all user registerd for a meetup of data changes
+     * @param dbMeetup
+     */
+    private notifyChanges(dbMeetup) {
+        const notification: PushNotification =
+            new PushNotification(NotificationId.MEETUPS_DATA_CHANGED, 'Meetup with id :' + dbMeetup.id + ' changed');
+
+        WebSockets.notify([dbMeetup.owner], notification);
+        DBMeetupRequest.find({meetup: dbMeetup.id}).then((dbRequests) => {
+            var users: string[] = [];
+            dbRequests.map((dbRequest) =>{
+                users.push(dbRequest.participant.toString());
+            })
+            WebSockets.notify(users, notification);
```

Comment:
Muss hier 2 mal notify sein oder liessen sich user und owner in einem Array übergeben?

Reply:
erledigt!